### PR TITLE
fixing unnecessary changes in controller

### PR DESCRIPTION
**Pull request by danylohlynsky**
Fixing problem from request #736

### CODE REVIEW

**Review by Max877866 — APPROVED** [review]

File: core/src/main/java/greencity/controller/OrderController.java [hunk 1]
```diff
@@ -455,19 +455,22 @@ public ResponseEntity<List<GetCourierLocationDto>> getCourierLocations(
...
             response.sendRedirect(redirectionConfigProp.getGreenCityClient());
         }
         return ResponseEntity.status(HttpStatus.OK).build();
     }
 
-    private void notifyPaidOrder(String orderId) {
-        Optional<Order> orderOptional = orderRepository.findById(Long.valueOf(orderId.split("_")[0]));
-        if (orderOptional.isPresent()) {
-            notificationService.notifyPaidOrder(orderOptional.get());
+    private void notifyPaidOrder(PaymentResponseDto dto) {
```
Comment (Andrii-Yezenitskyi) [inline core/src/main/java/greencity/controller/OrderController.java]: I think it would be better to extract this method into NotificationService

**Review by yourromchikkkkk — APPROVED** [review]

File: core/src/main/java/greencity/controller/OrderController.java (hunk 1 above)
Comment (danylohlynsky) [reply core/src/main/java/greencity/controller/OrderController.java]: fixed

**Review by Pavl0-Gural — APPROVED** [review]

File: service/src/main/java/greencity/service/NotificationServiceImpl.java [hunk 2]
```diff
@@ -112,6 +112,17 @@ public void notifyPaidOrder(Order order) {
         sendNotificationsForBotsAndEmail(notification);
     }
 
+    @Override
+    public void notifyPaidOrder(PaymentResponseDto dto) {
+        if (dto.getOrder_id() != null) {
```
Comment (thesvider) [inline service/src/main/java/greencity/service/NotificationServiceImpl.java:117]: orderId

Comment (thesvider) [reply service/src/main/java/greencity/service/NotificationServiceImpl.java:117]: you can use @JsonProperty("order_id")

**Pull request merged**